gitgraph

```diff
@@ -252,6 +255,8 @@ function main()
         then
             dia "$TMP_SVG"
             rm "$TMP_SVG"
+        else
```

Comment:
I have no time for more work. You can continue my branch. I would appreciate if my commits were to enter for the Git history in some way.